### ISSUE TITLE
'None' for gradient Keras

### ISSUE THREAD

**Issue by radu941208** [keras-team/keras#13593]
The [keras.preprocessing](https://github.com/keras-team/keras-preprocessing) module has been splitted from the main Keras repository.

My Model:

![image](https://user-images.githubusercontent.com/26097066/69885723-243aef00-12a4-11ea-8f23-34891fed086c.png)

Input X is : [700,100] and y is [700,1]

I am getting this error:

![image](https://user-images.githubusercontent.com/26097066/69885752-47659e80-12a4-11ea-8091-e16d6c83c137.png)

I imported 2 embedding matrix, one from glove and from my own dataset, and I am trying to train the model on X.
Can anyone help with the error above?

**Comment by DuyguA** [comment]
I didn't get this model at all, do you want to make an Embedding layer into the middle of the model???
Embedding layer can only be the first layer. No wonder this graph does not compile.

If you want to refine the word embeddings on your own dataset, use `trainable=True` . Convert first Embedding layer to:

```
Embedding(vocab_size, 300, input_length=...., weights=Constant(embedding_matrix),trainable=True)
```
